lambda_code/src/fhir_repository.py

```diff
@@ -68,6 +78,42 @@ def delete_immunization(self, imms_id: str) -> dict:
                 raise UnhandledResponseError(message=f"Unhandled error from dynamodb: {e.response['Error']['Code']}",
                                              response=e.response)
 
+    def find_immunizations(self, nhs_number: str, disease_code: str):
+        """it should find all patient's Immunization events for a specified disease_code"""
+        # TODO:
```

Comment:
All of these need to be done before this is merged.

Reply:
No we have some of the answers. There are still things that should be left there. I'll update the comment.